docs/examples.rst

```diff
@@ -0,0 +1,74 @@
+Examples
+========
+
+SAMPL models
```

Comment:
I really like this style! This in-line example is much better than the raw examples we currently have :)